core/capabilities/ccip/delegate.go

```diff
@@ -155,13 +156,29 @@ func (d *Delegate) ServicesForSpec(ctx context.Context, spec job.Job) (services
 	// since all queries are scoped by config digest.
 	ocrDB := ocr2.NewDB(d.ds, spec.ID, 0, d.lggr)
 
-	homeChainContractReader, ccipConfigBinding, err := d.getHomeChainContractReader(
-		ctx,
-		homeChainRelayer,
-		spec.CCIPSpec.CapabilityLabelledName,
-		spec.CCIPSpec.CapabilityVersion)
+	var (
+		homeChainContractReader types.ContractReader
+		ccipConfigBinding       types.BoundContract
+	)
+	err = retry.Do(func() error {
+		var err2 error
+		homeChainContractReader, ccipConfigBinding, err2 = d.getHomeChainContractReader(
+			ctx,
+			homeChainRelayer,
+			spec.CCIPSpec.CapabilityLabelledName,
+			spec.CCIPSpec.CapabilityVersion)
+		return err2
+	},
+		retry.Attempts(0), // retry forever
+		retry.Delay(10*time.Second),
```

Comment:
nit but Could we move that to the const? I think it would be easier to read

Reply:
Thanks. And yes, definitely can be tested.